.github/workflows/ci.yaml

```
@@ -33,32 +33,10 @@ jobs:
       REAPER_ENCRYPTION_KEY: SECRET_KEY
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
-      - name: Setup Maven Cache
-        uses: actions/cache@v2
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven3-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven3-
-
-      - name: Setup Java
-        uses: actions/setup-java@v2
-        with:
-          distribution: 'adopt' # See 'Supported distributions' for available options
-          java-version: '11'
-
-      - name: Setup Node
-        uses: actions/setup-node@v1
-        with:
-          node-version: '10.x'
-
-      - name: Setup Python
-        uses: actions/setup-python@v1
-        with:
-          python-version: '3.x'
-          architecture: 'x64'
+      - name: Setup
+        uses: ./github/actions/setup
```

```suggestion
        uses: .github/actions/setup
```

CI is failing because it's complaining that the action cannot be found.
Most probably because it's under `.github` instead of `./github`.

---

Sorry for the late reply.
Sorry for the typo.
I will upload the corrected version here soon.

---

Fix completed.
https://github.com/thelastpickle/cassandra-reaper/compare/1f394965fdc0facc260c0a75a5827f7840c3ce75..558cb6f148ff3807f5cf513037597a1ffd0a3961

---

@adejanovski 
Please check again.